packages/klighd-core/src/views-rendering.tsx

```diff
@@ -266,6 +255,41 @@ export function renderRectangularShape(rendering: KContainerRendering, parent: S
         }
     }
 
+    if (element && context.depthMap) {
+        const region = context.depthMap.getRegion((parent as KNode).id)
+        if (region && region.expansionState !== Visibility.Expanded && !region.hasTitle) {
+            // Render indirect region titles.
+            if (region.superStateTitle) {
+                const titleSVG = renderKText(region.superStateTitle, region.boundingRectangle, propagatedStyles, context, mListener)
+                element.children ? element.children.push(titleSVG) : element.children = [titleSVG]
+            } else if (region.macroStateTitle && !region.hasMultipleMacroStates) {
+                const titleSVG = renderKText(region.macroStateTitle, region.boundingRectangle, propagatedStyles, context, mListener)
+                element.children ? element.children.push(titleSVG) : element.children = [titleSVG]
+                // If there is no title draw placeholder.
```

Comment:
I think the placeholder could be rendered always, if there is further content to be expanded, otherwise this seems insonsistent if only some regions have that symbol.

Reply:
I believe this should be resolved by Mikas latest changes